content/en/docs/user-guides/troubleshooting.md

```diff
@@ -10,6 +10,11 @@ To suggest a troubleshooting item, [create a pull request PR with your suggested
 {{% /alert %}}
 
 
+## Where is the Notation CLI installation/configuration files/certificates/trust store?
+
+By default, the Notation CLI stores configuration files, certificates, and trust stores in the `NOTATION_BIN`, `NOTATION_LIBEXEC`, and `NOTATION_CONFIG` directories, which are specific to the operating system. For more details, see [Notation directory structure for system configuration]({{< ref "/docs/user-guides/how-to/directory-structure" >}}).
```

Comment:
One of the problems that I have with this statements is that they set the expectation that those environment variables are set as part of the installation, which is misleading. For example, if I install Notation with Brew, I don't have those set and cannot query them.

Example:
```
toddysm@tsm-mbp-msft  ~  env | grep NOTATION           
 ✘ toddysm@tsm-mbp-msft  ~  
```

Reply:
Good call @toddysm 

I updated the PR to make this clearer.